app/src/PostDetail/index.js

```diff
@@ -22,22 +22,16 @@ const PostDetail = () => {
     [id],
   );
 
-  const addComment = (comment) =>
-    apiClient.addComment(comment).then(getComments);
-
   React.useEffect(() => {
     id !== undefined && getPost() && getComments();
   }, [id, getPost, getComments]);
 
   const onSubmit = (event) => {
     const form = event.currentTarget;
-    const {
-      content: { value: content },
-      author: { value: author },
-    } = form.elements;
+    const comment = Object.fromEntries(new FormData(form).entries());
```

Comment:
@xiaozhong21 @tabbykatz I'd suggest using this way of extracting values from a form input, rather than the fancy destructuring. It's a lot more clear what's happening.